_includes/help-modal.html

```diff
@@ -57,12 +57,20 @@
 /* Modal Content/Box */
 .help-modal-content {
   background-color: #fefefe;
-  margin: 5% auto; /* 5% from the top and centered */
+  margin: 2% auto; /* 5% from the top and centered */
   border: 1px solid #888;
-  width: 60%; /* Could be more or less, depending on screen size */
-  @media (max-width: 768px) {
+  width: 55%; /* Could be more or less, depending on screen size */
+  @media (max-width: 480px) {
     margin: 25% auto;
     width: 85%; /* Change width to 85% */
+  };
+  @media (min-width: 481px) and (max-width: 768px) {
```

Comment:
why use of lot of media queries?

Reply:
To make it visible in most of the screens.

Reply:
Is this issue valid with you?

Reply:
this much of use of media quries doesn't sound reasonable. Its hard to maintain . These media quries is always hard to manually maintain. There is other css alternative also

Reply:
Noted